working-migration-guides/unreleased/bevy_render_2d.md

```diff
@@ -0,0 +1,33 @@
+# Create `bevy_render_2d` crate
+
+prs = [[18467](https://github.com/bevyengine/bevy/pull/18467)]
```

Comment:
Hi! The template now [is slightly different](https://github.com/bevyengine/bevy/blob/main/release-content/migration_guides_template.md) from Alice's PR, the title and related PRs must go in the ["front-matter"](https://jekyllrb.com/docs/front-matter/).

Also, the file should live in: `/release-content/migration-guides/`

```suggestion
---
title: Create `bevy_render_2d` crate
pull_requests: [18467]
---
```